weatherreport.cpp

```diff
@@ -20,22 +20,38 @@ namespace WeatherSpace
     /// without needing the actual Sensor during development
     /// </summary>
     class SensorStub : public IWeatherSensor {
+
+    public:
+        SensorStub() : temperatureInC(0.0),
```

Comment:
Good approach to parametrize. Reduces the need to write new classes for each situation. Can take these parameters in the constructor itself, it may reduce the test code further.

Reply:
Thank you. I have updated the code as per your suggestions.